### PR TITLE
Update 'block' and 'blocks' according to the latest changes

### PR DESCRIPTION
**Pull request by DarkEld3r**

### CODE REVIEW

File: src/advanced/node-management.md [hunk 1]
```diff
@@ -670,6 +672,7 @@ A JSON object with the following fields:
         }
       ],
       "txs": []
```
Comment (slowli) [inline src/advanced/node-management.md]: Missing comma after `[]`.

File: src/advanced/node-management.md [hunk 2]
```diff
@@ -609,9 +609,11 @@ A JSON object with the following fields:
 - **block**: BlockHeader  
   The header of the specified block
 - **precommits**: Array<Precommit\>  
-  The list of 'Precommit' messages supporting the block.
+  The list of 'Precommit' messages supporting the block
```
Comment (slowli) [inline src/advanced/node-management.md]: Nit: Please use backticks (\`Precommit\`) for code formatting.

File: src/advanced/node-management.md [hunk 3]
```diff
@@ -689,21 +692,30 @@ smallest and largest heights traversed to collect at most `count` blocks.
...
-equal than `start` and less than `end`.
+A JSON object with the following fields:
+
+- **range**: Range
+  The range specifies the largest and the smallest heights of blocks that have
+  been traversed to collect at most `count` blocks. The largest height `end`
+  equals to `latest + 1` if provided or to the height of the latest block in the
+  blockchain, the smallest height `start` takes values in `0..latest - count + 1`
+- **blocks**: Array<BlockHeader\>
+  The list of 'BlockHeader' in the requested range. Blocks in the array are sorted
```
Comment (slowli) [inline src/advanced/node-management.md]: The list of `BlockHeader`s...

**Comment by aleksuss** [comment]
Outdated

**Pull request closed without merging**